spec/formatting.md

```diff
@@ -319,7 +319,7 @@ the following steps are taken:
 
 #### Option Resolution
 
-The result of resolving _option_ values is a mapping of string identifiers to values.
+The result of resolving _option_ values is an unordered mapping of string identifiers to values.
```

Comment:
This might be too strong. An implementation would not be incorrect to order the options internally--that would be an implementation detail. What's important is that functions cannot count on the ordering or make the ordering significant.

Reply:
This has that effect. It's still possible for an implementation to use a container that retains order to represent the options, but with this qualifier functions can't rely on that.